src/App.js

```diff
@@ -1,9 +1,19 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components";
+import { Home, VideoListing } from "./pages";
+import { Route, Router, Routes } from 'react-router-dom';
+// import { Home, VideoListing } from "./pages";
```

Comment:
The commented line is already used in `line 3`, you may delete them from here

Reply:
okay, will do